docs/CompatibleSoftware.rst

```diff
@@ -19,63 +19,191 @@ application, please contact ocio-dev as well.
 After Effects
 *************
 
-`Compositor - Adobe <http://www.adobe.com/products/aftereffects.html>`__
+Website: `<http://www.adobe.com/products/aftereffects.html>`__
 
-An OpenColorIO plugin is available for use in After Effects.
+Documentation :
 
-See `src/aftereffects
-<http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/aftereffects>`__
-if you are interested in building your own OCIO plugins.
+- See `src/aftereffects <http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/aftereffects>`__ if you are interested in building your own OCIO plugins.
 
-Pre-built binaries are also available for `download
-<http://www.fnordware.com/OpenColorIO>`__, courtesy of 
-`fnordware <http://www.fnordware.com>`__.
+- Pre-built binaries are also available for `download <http://www.fnordware.com/OpenColorIO>`__, courtesy of `fnordware <http://www.fnordware.com>`__.
+
+
+- Profile has been properly measured using a spectrophotometer), then choose your display. If the transform was approved on a different monitor, then maybe you
```

Comment:
This third bullet point doesn't make sense. It can be deleted altogether likely.